.github/workflows/benchmark-runner.yml

```diff
@@ -45,6 +45,8 @@ jobs:
     needs: start-runner
     runs-on: ${{ needs.start-runner.outputs.runner_label }}
     steps:
+      - name: Pre checkout deps
+        run:  sudo apt-get update && sudo apt-get -y install git
```

Comment:
We always use Ubuntu/Debian here, right?

Reply:
as far as I know...